docs/configuration/git-providers/github-enterprise.md

```diff
@@ -1,5 +1,11 @@
 # GitHub Enterprise
 
+## Create a GitHub App
```

Comment:
I was not doing GitHubEnterprise in this pr.
If we want to do both I have to update https://bitbucket.org/qamine/codacy-website/pull-requests/3255/feature-move-github-self-hosted/diff

Reply:
Ah, yes, I know. These changes already came from `master` (originally done here: https://github.com/codacy/chart/pull/346).

Can we simply assume that the docs on how to configure the Git providers are unstable for now and I'll stabilize them in the next couple of days?

Reply:
Even so, the changes for GitHub Enterprise done on https://github.com/codacy/chart/pull/346 were just to take into account GitHub Apps authentication. They do not reflect the configuration of the Client ID and Client Secret on the .yaml file side yet.

Reply:
When this is merged to master this is published?
If yes we should stop it.
Docs should be published for a release, or hotfix.
We should not have docs that do not match the real case.

Reply:
I agree with you @rtfpessoa.

To clarify:
* When you opened this PR, the documentation was heavily out of date and had not been updated to take into account GitHub Apps authentication yet.
* While this PR was already ongoing, @mrfyda updated the documentation  on `master` to take into account GitHub Apps (PR https://github.com/codacy/chart/pull/346).
* Because it's easier to make these updates on moving the configurations to .yaml files on top of a more up-to-date base documentation, I merged `master` to this branch (https://github.com/codacy/chart/pull/348/commits/c44972f71092222ec14d0c6c04d11b9b3d3fc8cd). Otherwise, we would have tricky conflicts when we finally merged this PR back to `master`...
* After this merge I worked on updating the documentation to take into account the new configurations being done directly in .yaml files instead of the interface.

Which means we should be ready to go. :slightly_smiling_face: